plugin/eleline.vim

```diff
@@ -189,6 +189,8 @@ function! ElelineGitStatus() abort
   endif
   if max(l:summary) > 0
     return ' +'.l:summary[0].' ~'.l:summary[1].' -'.l:summary[2].' '
+  elseif !empty(get(b:, 'coc_git_status', ''))
+    return ' '.b:coc_git_status.' '
```

Comment:
hmm, what does `b:coc_git_status` look like? Does it return the same content as the others?

Reply:
yes it looks like this:

![image](https://user-images.githubusercontent.com/33961674/102314509-dc503e00-3f72-11eb-94a7-516fb4d40e75.png)